dev/flutter/todo/submissions/Dishal Kanishka/todo_application/lib/screens/view_todo.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:flutter/material.dart';
+import 'package:todo_application/Class/todo.dart';
+import 'package:todo_application/Constants/colors.dart';
+
+class TodoView extends StatefulWidget {
+  Todo todo;
+  TodoView({Key? key, required this.todo}) : super(key: key);
+
+  @override
+  _TodoViewState createState() => _TodoViewState(todo: this.todo);
+}
+
+class _TodoViewState extends State<TodoView> {
+  Todo todo;
+  _TodoViewState({required this.todo});
+  TextEditingController titleController = TextEditingController();
+  TextEditingController descriptionController = TextEditingController();
+
+  @override
+  void initState() {
+    super.initState();
+    if (todo != null) {
+      titleController.text = todo.title;
+      descriptionController.text = todo.description;
+    }
+  }
+
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Theme.of(context).canvasColor,
+      appBar: AppBar(
+        centerTitle: true,
+        elevation: 10,
+        backgroundColor: Theme.of(context).appBarTheme.backgroundColor,
+        title: const Text("View Your To Do"),
+      ),
+      body: SingleChildScrollView(
+        child: Container(
+          padding: const EdgeInsets.all(20),
+          child: Column(
+            children: [
+              Container(
+                child: colorOverride(
+                  TextField(
+                    onChanged: (data) {
+                      todo.title = data;
+                    },
+                    style: TextStyle(color: Theme.of(context).primaryColor),
+                    decoration: InputDecoration(
```

Comment:
The input here and below are of the same style, we could move this into a reusable component instead of redefining the styles twice